examples/core/gated_gcn/model.py

```diff
@@ -0,0 +1,58 @@
+"""
+Gated Graph Convolutional Network module for graph classification tasks
+"""
+import torch.nn as nn
+import torch.nn.functional as F
+from dgl.nn.pytorch import GatedGCNConv
+from dgl.nn.pytorch.glob import AvgPooling
+from ogb.graphproppred.mol_encoder import AtomEncoder, BondEncoder
+
+
+class GatedGCN(nn.Module):
+    def __init__(
+        self,
+        hid_dim,
+        out_dim,
+        num_layers,
+        dropout=0.2,
+        batch_norm=True,
+        residual=True,
+        activation=F.relu,
```

Comment:
You probably don't need to expose the arguments with L16-20. Just use a default value below.